digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/GeometryUtils.scala

```diff
@@ -194,5 +215,18 @@ object GeometryUtils {
     ((projection.oldEnd - projection.oldStart)*(projection.newEnd - projection.newStart)) < 0
   }
 
+  /**
+    * Creates a pseudo VVH Timestamp for new assets and speed limits. Turns clock back to 0:00 on the same day
+    * if less than offsetHours have passed since or 0:00 on previous day if not.
+    *
+    * @param offsetHours Number of hours since midnight to return current day as a VVH timestamp (UNIX time in ms)
+    */
+  def createVVHTimeStamp(offsetHours: Int): Long = {
+    val oneHourInMs = 60 * 60 * 1000L
+    val utcTime = DateTime.now().minusHours(offsetHours).getMillis
+    val curr = utcTime + DateTimeZone.getDefault.getOffset(utcTime)
+    curr - (curr % (24L*oneHourInMs))
+  }
+
```

Comment:
Maybe it's a good idea change the method createVVHTimeStamp to the VVClient instead of be in GeometryUtils